docs/source/quicktour.mdx

```diff
@@ -14,186 +14,186 @@ specific language governing permissions and limitations under the License.
+**Audio**:
+* Audio classification: assign a label to a given segment of audio.
+* Automatic speech recognition (ASR): transcribe audio data into text.
+
+<Tip>
+For more details about the [`pipeline`] and associated tasks, refer to the documentation [here](./main_classes/pipelines)
+</Tip>
 
-Install the following dependencies (if not already installed):
+## Using the pipeline
+
+In the following example, we will look at how to use the [`pipeline`] for sentiment analysis.
+
+Install the following dependencies if you haven't already:
 
 ```bash
 pip install torch
 ===PT-TF-SPLIT===
 pip install tensorflow
 ```
 
-```py
+Import the [`pipeline`] with the task you want to complete:
+
+```python
 >>> from transformers import pipeline
->>> classifier = pipeline('sentiment-analysis')
+>>> classifier = pipeline("sentiment-analysis")
 ```
 
-When typing this command for the first time, a pretrained model and its tokenizer are downloaded and cached. We will
-look at both later on, but as an introduction the tokenizer's job is to preprocess the text for the model, which is
-then responsible for making predictions. The pipeline groups all of that together, and post-process the predictions to
-make them readable. For instance:
+The pipeline downloads and caches a default [pretrained model](https://huggingface.co/distilbert-base-uncased-finetuned-sst-2-english) and tokenizer for sentiment analysis. Now you can use the `classifier` on your target text:
 
-
-```py
->>> classifier('We are very happy to show you the 🤗 Transformers library.')
-[{'label': 'POSITIVE', 'score': 0.9998}]
+```python
+>>> classifier("We are very happy to show you the 🤗 Transformers library."")
+[{"label": "POSITIVE", "score": 0.9998}]
 ```
 
-That's encouraging! You can use it on a list of sentences, which will be preprocessed then fed to the model, returning
-a list of dictionaries like this one:
+For more than one sentence, pass a list of sentences to the [`pipeline`] which returns a list of dictionaries:
 
-```py
+```python
 >>> results = classifier(["We are very happy to show you the 🤗 Transformers library.",
 ...            "We hope you don't hate it."])
 >>> for result in results:
-...     print(f"label: {result['label']}, with score: {round(result['score'], 4)}")
+...     print(f"label: {result["label"]}, with score: {round(result["score"], 4)}")
 label: POSITIVE, with score: 0.9998
 label: NEGATIVE, with score: 0.5309
 ```
 
-To use with a large dataset, look at [iterating over a pipeline](main_classes/pipelines)
+The [`pipeline`] can also iterate over an entire dataset. Start by installing the 🤗 Datasets library:
+
+```bash
+pip install datasets 
+```
+
+Create a [`pipeline`] with the task you want to solve for, and the model you want to use:
 
-You can see the second sentence has been classified as negative (it needs to be positive or negative) but its score is
-fairly neutral.
+```python
+>>> from transformers import pipeline
```

Comment:
We generally style doc samples to have one empty line between import and code (like black would do).